.releaserc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "branches": [{ "name": "main" }, { "name": "alpha", "prerelease": true }],
```

Comment:
Since you added other branches to the .github/workflows/release.yml, we need to declare them as prerelease here, too :)